src/Bouncer.php

```diff
@@ -248,10 +248,6 @@ public function gate()
             return $this->gate;
         }
 
-        if ($throw) {
-            throw new RuntimeException('The gate instance has not been set.');
```

Comment:
Why did you remove this throw?

Reply:
Hello,
This throw will never happen because a few commits back the variable was removed from the method, and is always null:
    /**
     * Get the gate instance. Throw if not set.
     *
     * @return \Illuminate\Contracts\Auth\Access\Gate
     *
     * @throws \RuntimeException
     */
    public function gate()
    {
        if ($this->gate) {
            return $this->gate;
        }
        return null;
    }
this is how the method looks now on the beta5, and it has no variable throw received as parameter.

Reply:
I understand why you removed the conditional. That was wrong :+1: 

I was asking why you removed the throw. I changed it here: https://github.com/JosephSilber/bouncer/commit/485dbe0007171e284e5ae99f1480c6bb6e54036e

Reply:
I thought since it was never actually thrown, it wasn`t needed, so I removed it completely. I guess it will never return null now as it used to, correct?

Reply:
This method is used internally, where certain methods in the `Bouncer` class need to call methods directly on the `Gate`. Instead of checking it each time, we check it here once.